frappe/patches/v8_0/update_gender_and_salutation.py

```diff
@@ -2,7 +2,7 @@
 
 from __future__ import unicode_literals
```

Comment:
Create a new patch for updating the Genders and Salutations rather than making changes to v8 patch